### PR TITLE
CI: Update Setup Java Action

### PR DESCRIPTION
**Pull request by ueman**
## :scroll: Description
setup-java had an update. So we update it:
https://github.com/actions/setup-java

_#skip-changelog_

## :pencil: Checklist

- [ ] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] I updated the docs if needed
- [ ] All tests passing
- [ ] No breaking changes

### CODE REVIEW

File: .github/workflows/flutter.yml [hunk 1]
```diff
@@ -32,8 +32,9 @@ jobs:
     steps:
         - uses: actions/checkout@v2
 
-        - uses: actions/setup-java@v1
+        - uses: actions/setup-java@v2
           with:
+            distribution: 'zulu'
```
Comment (marandaneto) [inline .github/workflows/flutter.yml]:
```suggestion
            distribution: 'adopt'
```

File: .github/workflows/flutter.yml [hunk 2]
```diff
@@ -124,8 +125,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-java@v1
+      - uses: actions/setup-java@v2
         with:
+          distribution: 'zulu'
```
Comment (marandaneto) [inline .github/workflows/flutter.yml]:
```suggestion
          distribution: 'adopt'
```

**Comment by marandaneto** [comment]
`adopt` is a bit more known, lets use this one :)

**Review by marandaneto — APPROVED** [review]

**Comment by ueman** [comment]
> `adopt` is a bit more known, lets use this one :)

Sure. I just choose `zulu` because it was the default in v1 🤷‍♂️

**Comment by marandaneto** [comment]
> > `adopt` is a bit more known, lets use this one :)
> 
> Sure. I just choose `zulu` because it was the default in v1 🤷‍♂️

yeah I know, just read their changelog and noticed it :D

**Pull request merged**